public/index.html

```diff
@@ -3,10 +3,12 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <meta name="theme-color" content="#000000">
+    <meta name="theme-color" content="#e95420">
```

Comment:
Our theme color is `#f96232`. Also is there a `background-color` meta as well? If so please add it with `white` as in the manifest.

Reply:
Thank you! I didn't know the exact value so I grabbed it from the website. Gonna update that too.

No, that's the only meta. There are a bunch of other one iOS specific but we can add them on the future